test/test-single-beat.sh

```diff
@@ -1,17 +1,21 @@
 #!/bin/bash
-(SINGLE_BEAT_REDIS_SERVER='redis://localhost:6379/9' SINGLE_BEAT_LOG_LEVEL=debug SINGLE_BEAT_IDENTIFIER='test-beat' single-beat python long_waiting_process.py > first_single_beat.log) &
+
+rm -f first_single_beat.log \
+	second_single_beat.log \
+	third_single_beat.log
+
+(SINGLE_BEAT_REDIS_SERVER='redis://localhost:6379/9' SINGLE_BEAT_LOG_LEVEL=debug SINGLE_BEAT_IDENTIFIER='test-beat' python ../singlebeat/beat.py sh -c 'exec python long_waiting_process.py > first_single_beat.log') &
```

Comment:
We must move the redirection into the command passed to single-beat so that the subprocess can flush its output. See #28